packages/components/src/keyboard-shortcuts/index.native.js

```diff
@@ -1,2 +1,36 @@
-const KeyboardShortcuts = () => null;
+/**
```

Comment:
I think the web version of this file could work as well. Here we're creating another component with the same naming but which accepts different props. I'd rather see us stripping the modifier from the shortcut in `KeyboardShortcuts`.